integ-test/src/test/java/org/opensearch/sql/ppl/dashboard/templates/dashboard/cloudtrail.rst

```diff
@@ -0,0 +1,294 @@
+..
+   Copyright OpenSearch Contributors
+   SPDX-License-Identifier: Apache-2.0
+
+============================
+CloudTrail Dashboard Queries
+============================
```

Comment:
As I understand, the IT introduced in this PR is like sanity test in https://github.com/opensearch-project/opensearch-spark/pull/995. But could you clarify what's the purpose of this doc? What is the audience of this doc?

Reply:
I see. So this is essentially to showcase how to use PPL to do analytics on real-world data (AWS logs). I feel our RST is not the best place for this. Please follow up with @ritvibhatt for this.